update-dois/src/main/java/org/reactome/release/updateDOIs/Main.java

```diff
@@ -12,55 +13,62 @@
 //  final static Logger logger = Logger.getLogger(UpdateDOIs.class);
 	private static final Logger logger = LogManager.getLogger();
 
-  public static void main( String[] args ) {
+  public static void main( String[] args ) throws IOException {
 
 	 // Default locations of properties and pre-set report files
 	 // Will override if arguments are provided
 	 // TODO: Lots of room for error here if only 1 argument provided, or if the report file isn't updated
     String pathToConfig = "src/main/resources/config.properties";
     String pathToReport = "src/main/resources/UpdateDOIs.report";
+	  
+    boolean testMode = true;
     if (args.length > 0 && !args[0].equals(""))
     {
       pathToConfig = args[0];
-      pathToReport = args[1];
+      if (args.length > 1 && !args[1].equals("")) {
+        pathToReport = args[1];
+        testMode = false;
+      }
```

Comment:
A ternary for pathToConfig used at the time of declaration might be more succinct and easier to follow than a nested if:
```
String pathToConfig = args.length > 0 && !args[0].isEmpty() ? args[0] : Paths.get("src/main/resources/config.properties").toString();

String pathToReport = Paths.get("src/main/resources/UpdateDOIs.report").toString(); 
boolean testMode = true;
if (args.length > 1 && !args[1].isEmpty()) {
    pathToReport = args[1];
    testMode = false;
}
```

Reply:
Thanks, implemented